app/src/main/java/org/oppia/app/topic/overview/TopicOverviewFragmentPresenter.kt

```diff
@@ -59,9 +60,12 @@ class TopicOverviewFragmentPresenter @Inject constructor(
     })
   }
 
-  // TODO(#135): Get this topic-id from [TopicFragment].
   private val topicResultLiveData: LiveData<AsyncResult<Topic>> by lazy {
-    topicController.getTopic(TEST_TOPIC_ID_1)
+    topicId =
+      if (fragment.arguments != null && fragment.arguments!!.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) != null) fragment.arguments!!.getString(
+        TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY
+      ) else ""
```

Comment:
I think here and elsewhere this logic can be simplified to:
`topicId = fragment.arguments?.getString(TOPIC_ACTIVITY_TOPIC_ID_ARGUMENT_KEY) ?: ""`

Reply:
> @nikitamarysolomanpvt
> 
> 1. The issue that you have mentioned related to Topic-Play has been fixed and is merged with develop.
> 2. Please resolve these merge conflicts and make this branch up-to-date.

Okay thank you.

Reply:
Resolved